app/views/docs/queries.phtml

```diff
@@ -0,0 +1,781 @@
+<p>Many list endpoints in Appwrite allow you to filter results using queries. Appwrite provides a shared set of query syntax to allow filtering, pagination, and sorting of results. When using an Appwrite SDK, a <code>Query</code> class is provided to help you build queries.</p>
+
+<h2><a href="/docs/queries#query-compatibility" id="queryable-compatibility">Query compatibility</a></h2>
+<p>Each list endpoint supports different query operations. In Appwrite Databases, querying documents by attributes require an <a href="/docs/databases#indexes">index</a> on the attributes. In other services, only certain attributes are indexed and can be queried. You can find the supported query methods and attributes in the <b>References</b> section of the Appwrite documentation.</p>
+
+<h2><a href="/docs/queries#query-class" id="query-class">Query Class</a></h2>
+<p>Appwrite SDKs provide a <code>Query</code> class to help you build queries. The <code>Query</code> class has a method for each type of supported query.</p>
+<table cellspacing="0" cellpadding="0" border="0" class="full margin-bottom-large">
+    <thead>
+        <tr>
+            <th style="width: 100px">Query Method</th>
+            <th>SDK Method Example</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>Equal</td>
+            <td><code>Query.equal("title", ["Iron Man"])</code></td>
+            <td>Returns document if attribute is equal to any value in the provided array. Applies to any indexed attribute.</td>
+        </tr>
+        <tr>
+            <td>Not Equal</td>
+            <td><code>Query.notEqual("title", ["Iron Man"])</code></td>
+            <td>Returns document if attribute is not equal to any value in the provided array. Applies to any indexed attribute.</td>
+        </tr>
+        <tr>
+            <td>Less Than</td>
+            <td><code>Query.lessThan("score", 10)</code></td>
+            <td>Returns document if attribute is less than the provided value. Applies to any indexed attribute.</td>
+        </tr>
+        <tr>
+            <td>Less Than or Equal</td>
+            <td><code>Query.lessThanEqual("score", 10)</code></td>
+            <td>Returns document if attribute is less than or equal to the provided value. Applies to any indexed attribute.</td>
+        </tr>
+        <tr>
+            <td>Greater Than</td>
+            <td><code>Query.greaterThan("score", 10)</code></td>
+            <td>Returns document if attribute is greater than the provided value. Applies to any indexed attribute.</td>
+        </tr>
+        <tr>
+            <td>Greater Than or Equal</td>
+            <td><code>Query.greaterThanEqual("score", 10)</code></td>
+            <td>Returns document if attribute is greater than or equal to the provided value. Applies to any indexed attribute.</td>
+        </tr>
+        <tr>
```

Comment:
We now also have 
- `isNull`: Check if an attribute value is null
- `isNotNull`: Check if an attribute value is notnull
- `startsWith`: Check if a string attribute starts with another string
- `endsWith`: Check if a string attribute ends with another string
- `between`: Check if a value falls in a range. Applies to int, float, date, string e.g. `Query.between("price", 5, 10)`;
- `select`: Fetch only a specific subset of attributes of a document e.g. `Query.select(['name', 'title'])`